tests/js/package.json

```diff
@@ -5,25 +5,23 @@
 		"test:js:watch": "npm run test:js -- --watch"
 	},
 	"dependencies": {
-		"@googlesitekit/assets": "file:../../assets"
+		"@googlesitekit/assets": "file:../../assets",
+		"@wordpress/element": "^2.20.3",
```

Comment:
Isn't element a dependency of `assets` already?

Reply:
It is, but we have a direct import of `@wordpress/element` in one of `tests/js` files. So, it is better to have this package added directly to the dependencies list instead of relying on it indirectly.